airflow/providers/google/provider.yaml

```diff
@@ -151,7 +151,7 @@ additional-extras:
       - apache-beam[gcp]
   - name: cncf.kubernetes
     dependencies:
-      - apache-airflow-providers-cncf-kubernetes>=7.2.0
+      - apache-airflow-providers-cncf-kubernetes>=7.14.0
```

Comment:
I think it's the cleanest way, otherwise the logic will be a bit complicated (check my previous commit)

Reply:
I am OK with bumping the min version